docs/UserGuide.md

```diff
@@ -193,17 +199,23 @@ in their address.
 <br><br/>
 * The search keywords used are case-insensitive. e.g. `keychain` will match `Keychain`
 * The order of the keywords does not matter. e.g. `apple keychain` will match `Keychain Apple`
-* Only full words will be matched e.g. `keychains` will not match `keychain`
+* Only full words will be matched e.g. `Gardens,` will not match `Gardens` and `keychain` will not match `keychains`
 * Orders matching at least one keyword will be returned (i.e. `OR` search).<br>
```

Comment:
Would suggest removing the ``OR` search` portion. Might be confusing to users that don't know boolean logic